addons/escoria-core/game/esc_autoload.gd

```diff
@@ -75,8 +78,8 @@ var dialog_player: ESCDialogPlayer
 # Inventory scene
 var inventory
 
-# These are settings that the player can affect and save/load later
-var settings: ESCSaveSettings
+## These are settings that the player can affect and save/load later
+#var settings: ESCSaveSettings
```

Comment:
Since this is Git, let's remove commented variables and code since we can always pull the history. It keep things cleaner.

Reply:
Done